integration/ct_functions.sh

```diff
@@ -0,0 +1,90 @@
+# Functions for setting up CT personalities in Trillian integration tests
+# Requires github.com/google/trillian/integration/functions.sh
+
+declare -a CT_SERVER_PIDS
+CT_SERVERS=
+CT_CFG=
+
+# ct_prep_test prepares a set of running processes for a CT test.
+# Parameters:
+#   - number of log servers to run
+#   - number of log signers to run
+#   - number of CT personality instances to run
+# Populates:
+#  - CT_SERVERS     : list of HTTP addresses (comma separated)
+#  - CT_SERVER_PIDS : bash array of CT HTTP server pids
+# in addition to the variables populated by logPrepTest.
+ct_prep_test() {
+  # Default to one of everything.
+  local RPC_SERVER_COUNT=${1:-1}
```

Comment:
nit: Variables should be lower_camel, UPPER_CAMEL is for globals.

https://google.github.io/styleguide/shell.xml#Variable_Names